src/app/Rules/CreateRule.tsx

```diff
@@ -73,7 +74,7 @@ import { HelpIcon } from '@patternfly/react-icons';
 import _ from 'lodash';
 import * as React from 'react';
 import { useHistory, withRouter } from 'react-router-dom';
-import { forkJoin, iif, of, Subject } from 'rxjs';
```

Comment:
Hey @aali309, can u back out these rearragement of imports? Just to keep it clean.

Reply:
Same for `src/app/Shared/MatchExpression/MatchExpressionVisualizer.tsx`.